client/a8c-for-agencies/sections/marketplace/products-overview-v2/product-filter/style.scss

```diff
@@ -68,7 +68,7 @@
 		box-shadow: inset 0 0 0 1px var(--color-neutral-20);
 		color: var(--color-accent-60);
 		fill: var(--color-accent-60);
-		background: rgba(var(--wp-admin-theme-color--rgb), 0.04);
+		background: color-mix(in srgb, var(--wp-admin-theme-color-) 4%, transparent);
```

Comment:
this is broken

Reply:
Thanks, should we be replacing the wp-admin variables? Might as well?

Reply:
Sounds good to keep it the same as before, except that Gutenberg also plans to remove the RGB variable 🤔 But either is fine.

Reply:
Fixed in 0648bf0